pom.xml

```diff
@@ -9,7 +9,7 @@
     <artifactId>prestogateway-parent</artifactId>
     <name>prestogateway-parent</name>
     <packaging>pom</packaging>
-    <version>1.6.9</version>
+    <version>1.6.10</version>
```

Comment:
Nit: can you bump the version to 1.7.0? i think this warrants a minor version bump